client/src/app/tabs/getEditMenu.js

```diff
@@ -254,6 +256,16 @@ export function getSelectionEntries({
     });
   }
 
+
+  if (isDefined(replaceElement)) {
+    menuEntries.push({
+      label: 'Replace element',
+      accelerator: 'R',
+      enabled: replaceElement,

```

Comment:
When I select multiple elements, the replace shortcut is enabled but has no effect:

![Recording 2023-01-02 at 16 07 25](https://user-images.githubusercontent.com/21984219/210249232-2b46ec72-d394-48e3-9fb6-3ed905594f6e.gif)

IMO, this should not block us if we can't fix it right now. We are inconsistent with this with other shortcuts as well (e.g. "edit label"). We should consider opening a follow-up issue to track it if we decide to not fix it right now.